Index: src/main/java/kr/com/study/자료구조/이진탐색/이진탐색트리.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/kr/com/study/자료구조/이진탐색/이진탐색트리.java b/src/main/java/kr/com/study/자료구조/이진탐색/이진탐색트리.java
new file mode 100644
--- /dev/null	(date 1722136172569)
+++ b/src/main/java/kr/com/study/자료구조/이진탐색/이진탐색트리.java	(date 1722136172569)
@@ -0,0 +1,7 @@
+package kr.com.study.자료구조.이진탐색;
+
+public class 이진탐색트리 {
+
+
+
+}
